source/05-rspec/02-matchers.md

```diff
@@ -16,22 +16,22 @@ expect(user.born_in_leap_year?).to eq true
 What does `expect` do, exactly? And what's the deal with `to` and `eq`?
 
 These also are methods that RSpec defines for us, so we can describe our
-expectations in a readable way. I.e. these methods are RSpec's equivalent to
+expectations in a readable way. That is to say, these methods are RSpec's equivalent to
 assertions (`assert` and friends) in Minitest.
 
 Technically, `expect` returns an object that responds to the method `to`. This
-method `to` expects to be passed an object that is a so called matcher. It will
+method `to` expects to be passed an object that is a so-called matcher. It will
 then call the matcher to see if it ... well, matches.
 
-Remember that in Ruby you can omit parentheses when calling a method. I.e. we
-could also add them:
+Remember that in Ruby you can omit parentheses when calling a method (as long as this doesn't make your code ambiguous). So we
+could just as well add them:
 
 ```ruby
-expect(user.born_in_leap_year?).to eq(true)
+expect(user.born_in_leap_year?).to(eq(true))
```

Comment:
I added these parentheses because I thought you actually wanted to provide the version that didn't leave out _any_(?)